mudlet-mapper.xml

```diff
@@ -10877,6 +10884,62 @@ end
 					</eventHandlerList>
 				</Script>
 			</Script>
+			<Script isActive="yes" isFolder="no">
+				<name>map features</name>
+				<packageName></packageName>
+				<script>-------------------------------------------------
+--         Put your Lua functions here.        --
+--                                             --
+-- Note that you can also use external Scripts --
+-------------------------------------------------
+
+local function loadMapFeatures()
+  local mapFeaturesString = getRoomUserData(1, "mapFeatures")
```

Comment:
We should just use map user data for this now, since it's available

Reply:
Looking at the docs, the required map format is available since 3.0.1, which is old enough I guess. I thought it was a more recent addition.